community/index/src/main/java/org/neo4j/index/internal/gbptree/GBPTree.java

```diff
@@ -757,9 +765,12 @@ private void close( Header.Writer headerWriter ) throws IOException
 
         try
         {
-            // Perform a checkpoint before closing. If no changes has happened since last checkpoint,
-            // no new checkpoint will be created.
-            checkpoint( IOLimiter.unlimited(), headerWriter );
+            if ( health.isHealthy() )
```

Comment:
Ye, it would be good. It should to be logged from `LabelScanStore` level though as `GBPTree` is just some underlying implementation detail.

It is not super clear to be how this should be with relation to `LuceneLabelScanStore`.

But maybe something like:
- Add `close( didCheckpoint )` to `GBPTree.Monitor` and trigger during `GBPTree.close()`.
- Propagate monitor call to `LabelScanStore.Monitor` which register it as `shutdown( didCheckpoint )`. 
- `LoggingMonitor` implementation log some line about this.

Problem: What does `LabelScanStore.shutdown( didCheckpoint )` mean for `Lucene` implementation?